src/oceans/models/guide.js

```diff
@@ -479,7 +10,9 @@ const getCurrentGuide = () => {
 
   const state = getState();
 
-  for (const guide of guides) {
+  const currentGuides = queryStrFor('guides') === 'K5' ? guidesK5 : guidesHoc;
```

Comment:
I assume this will be configurable on the studio side?

Reply:
Great question.  I just updated this PR to make this a parameter for `initAll`, and then we will be able to configure this on the studio side.